CodedEvents/backend/models.py

```diff
@@ -1,3 +1,56 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Role(models.Model):
+    name = models.CharField(max_length=30)
+    desc = models.CharField(max_length=100)
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    roles = models.ManyToManyField(Role)
+
+
+class Address(models.Model):
+    street = models.TextField
+    city = models.TextField
+    province = models.TextField
+    code = models.TextField
+
+    class Meta:
+        verbose_name_plural = 'Addresses'
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=30)
+    desc = models.CharField(max_length=100)
+
+    class Meta:
+        verbose_name_plural = 'Categories'
+
+
+class Event(models.Model):
+    name = models.CharField(max_length=30)
+    desc = models.CharField(max_length=100)
+    organizers = models.ManyToManyField(Profile)
+    address = models.ForeignKey(Address, on_delete=models.CASCADE)
```

Comment:
convention-wise, i think it'll be advisable to have suffixes for foreign keys such as 'address_id', 'event_id' instead of 'event' and 'address'. Thoughts?

Reply:
Should we change the class names? Since I have to put the class names in that spot. I don't think we should have 'address_id' as a class name though?

Reply:
class names? I only meant the references to the fields in the db. 

e.g: Instead of 
address = models.ForeignKey(Address, on_delete=models.CASCADE)

we have:
address_id = models.ForeignKey(Address, on_delete=models.CASCADE)

Reply:
I see, sure